go/ql/lib/change-notes/2023-10-20-enclosing-callable-for-external-files.md

```diff
@@ -0,0 +1,4 @@
+---
+category: fix
+---
+* Fixed a bug where data flow nodes in files that are not in the project being analyzed (such as libraries) and are not contained within a function were not given an enclosing callable. Note that for files that are not contained within a function, the enclosing callable is considered to be the file itself. This may cause some minor changes to results.
```

Comment:
Two comments:
- "enclosing callable" sounds a lot like an implementation detail, which is unavoidable here I think, but then it probably makes sense to highlight that `Callable` is a type name
- I think you mean "nodes" instead of "files" in the second sentence?

```suggestion
* Fixed a bug where data flow nodes in files that are not in the project being analyzed (such as libraries) and are not contained within a function were not given an enclosing `Callable`. Note that for nodes that are not contained within a function, the enclosing callable is considered to be the file itself. This may cause some minor changes to results.
```